web-components/src/[sandbox]/sandbox.mock.ts

```diff
@@ -395,77 +393,3 @@ export const DraggableOptions = [
     registered: "Tuesday, December 1, 2015 6:44 AM"
   }
 ];
-
-export const ShortkeyTable: { config: TableAdvanced.Config; data: TableAdvanced.Data } = {
-  config: {
-    isStickyHeader: true,
-
-    rows: {
-      isDraggable: false,
-      selectable: "multiple"
-    },
-
-    cols: {
-      isResizable: true,
```

Comment:
@boiarchukk Is all this being removed by accident?

Reply:
@futurethang It was removed with purpose to cleanup mock data duplication for Advance Table component.